examples/siro_sandbox/README.md

```diff
@@ -1,165 +1,136 @@
+    - [Launch Commands](#launch-commands)
+      - [GUI-controlled humanoid and learned-policy-controlled Spot](#gui-controlled-humanoid-and-learned-policy-controlled-spot)
+    - [Controls](#controls)
+    - [Command-line Options](#command-line-options)
+      - [App State and use cases](#app-state-and-use-cases)
+      - [Hack to hide the skinned humanoid in the GUI viewport](#hack-to-hide-the-skinned-humanoid-in-the-gui-viewport)
+      - [Saving episode data](#saving-episode-data)
+      - [Debugging visual sensors](#debugging-visual-sensors)
+      - [Debugging simulator-rendering](#debugging-simulator-rendering)
+      - [GUI-controlled agents and free camera mode](#gui-controlled-agents-and-free-camera-mode)
+      - [First-person and third-person mode for GUI-controlled humanoid](#first-person-and-third-person-mode-for-gui-controlled-humanoid)
+      - [Can grasp/place area](#can-graspplace-area)
+      - [Disable episode end on collision](#disable-episode-end-on-collision)
+      - [Play episodes filter](#play-episodes-filter)
+      - [Saving episode data](#saving-episode-data-1)
+      - [Capturing Gfx-Replay Files](#capturing-gfx-replay-files)
+      - [Human-in-the-loop tutorial sequence](#human-in-the-loop-tutorial-sequence)
+      - [Testing BatchReplayRenderer](#testing-batchreplayrenderer)
+  - [Sandbox Tool Architecture](#sandbox-tool-architecture)
+  - [Remote HITL Evaluation (VR)](#remote-hitl-evaluation-vr)
+
+
+## Installation
+1. Install Habitat-lab using [instructions](../../README.md#installation).
+2. Install Habitat-sim [main branch](https://github.com/facebookresearch/habitat-sim).
+    * [Build from source](https://github.com/facebookresearch/habitat-sim/blob/main/BUILD_FROM_SOURCE.md), or install the [conda packages](https://github.com/facebookresearch/habitat-sim#recommended-conda-packages).
+    * Make sure to include Bullet physics, e.g. `python setup.py install --bullet`.
+3. Download required assets:
+    ```bash
+    python -m habitat_sim.utils.datasets_download \
+    --uids hab3-episodes habitat_humanoids hab_spot_arm ycb hssd-hab \
+    --data-path data/
+    ```
+
+## Usage
+
+### Launch Commands
+
+#### GUI-controlled humanoid and learned-policy-controlled Spot
+To launch GUI-controlled humanoid and random-policy-controlled (initialized with random weights) Spot in HSSD, run:
+```bash
 HABITAT_SIM_LOG=warning MAGNUM_LOG=warning \
 python examples/siro_sandbox/sandbox_app.py \
 --disable-inverse-kinematics \
 --never-end \
 --gui-controlled-agent-index 1 \
---cfg experiments_hab3/pop_play_kinematic_oracle_humanoid_spot_fp.yaml \
+--app-state rearrange \
+--cfg social_rearrange/pop_play.yaml \
 --cfg-opts \
+habitat.environment.iterator_options.cycle=False \
 habitat_baselines.evaluate=True \
 habitat_baselines.num_environments=1 \
 habitat_baselines.eval.should_load_ckpt=False \
-~habitat.task.measurements.agent_blame_measure
+habitat_baselines.rl.agent.num_pool_agents_per_type='[1,1]' \
```

Comment:
I assume you are still planning to clean up this readme later, right? E.g. in the polished version, we shouldn't have so many required config overrides on the command line, since we have yaml files for that.